docs/peggy/Peggy-Ceth-Account-Set.md

```diff
@@ -20,7 +22,9 @@ Similar to the account setting method, the transaction is priviledged and only t
 It will transfer the cEth from ethbridge module to an specific account.
 
 1. command format
-sifnodecli tx ethbridge rescue_ceth [cosmos-sender-address] [ceth_receiver_account] [ceth_amount]
+```
+sifnodecli tx ethbridge rescue_ceth $clp_admin_address $ceth-receiver-account $ceth_amount
```

Comment:
@ElliotFriedman is this correct? should the sender account be the $clp_admin_address or $ethbridge-module address?

@ElliotFriedman do we need -from=$clp_admin_moniker or --from=$clp_admin_address too or smth?

Reply:
@musnit as the original documentation says, you need to put the tx sender as the first param to the rescue_ceth command. The second param is the sifchain address of the user who is receiving the ceth. The third param is the amount of ceth you are trying to rescue from the ethbridge module. You can only rescue the amount locked there.